src/esp/gfx/RenderTarget.h

```diff
@@ -24,6 +25,36 @@ namespace gfx {
  */
 class RenderTarget {
  public:
+  enum class Flag {
+    /**
+     * @brief No textures for the meshes.
+     * Textures take a lot of GPU memory but they are not needed unless we are
+     * doing RGB rendering.
+     * Note: Do NOT set this flag when doing RGB rendering
+     * Currently it is just used to track whether or not @ref readFrameRgba,
+     * @ref blitRgbaToDefault, and @readFrameRgbaGPU are valid calls.
+     */
+    NoTextures = 1 << 0,
```

Comment:
Doesn't `NoRgbaBuffer` imply `NoTextures`? I don't see a use case for one being set but the other not.

Reply:
OK.
First, I just inherited what was there before (the "NoTextures" item).
Second, "NoTextures" honestly does not lead to "NoRgbaBuffer" because in my opinion user can still render the mesh (with no textures) to a "rgbaBuffer" using the flat shaders.

Reply:
@erikwijmans : May I have your input? Actually this is a question I had for quite a long time.
If the textures are not loaded, the mesh can still be rendered using flat shader, and thus a RgbaBuffer is still needed (and can even be blit to the default FBO)?

Reply:
If there are no textures it doesn't make sense to render rgb IMO -- which is why there is that runtime_error if you try to read the rgb buffer when there are no textures.  Currently NoTextures is only set when there isn't an RGB camera on any agent.